introduction/crowdfunding_this_book.md

```diff
@@ -1,6 +1,6 @@
-## この本へのクラウドファンディング {#crowdfunding-this-book}
+# この本へのクラウドファンディング {#crowdfunding-this-book}
 
-私たちが読者のための素晴らしい本を作り続けたいなら、ピザ、寿司やコーヒーを買う必要がある。そのために十分なビットコインをいただくことに見合う責任が、私たちにはあるということでもある。また、私たちはとても怠け者なので、この本を書き続けるためにもぜひ読者のフィードバックをいただきたい。
+読者のために素晴らしい本を書き続けるためには、生きていくために食料品を買う必要があります。またこの本を書き、更新し続けることは寄付をいただくことに見合うことだとも考えています。さらに言えば、私たちはとても怠け者なので、この本を書き続けるためにもぜひ読者のフィードバックをいただきたい。
```

Comment:
```suggestion
読者のために素晴らしい本を書き続けるには、食料品を買う必要があります。また、そのための寄付を集める責任も我々には、あるとも考えています。さらに言えば、私たちはとても怠け者なので、この本を書き続けるためにもぜひ読者のフィードバックをいただきたい。
```

Reply:
@hypergori I actually want to avoid using `責任` here because according to Japanese dictionary, `責任` means

1. 自分が引き受けて行わなければならない任務。義務。
2. 自分がかかわった事柄や行為から生じた結果に対して負う義務や償い

`寄付を集める責任` doesn't make sense for me. What do you think?